azure/src/main/java/sunstone/azure/impl/AzureArchiveDeployerProvider.java

```diff
@@ -1,26 +0,0 @@
-package sunstone.azure.impl;
-
-
-import sunstone.azure.impl.AzureIdentifiableSunstoneResource.Identification;
-import sunstone.azure.annotation.AzureArchiveDeploymentAnnotation;
-import sunstone.core.AnnotationUtils;
-import sunstone.core.api.SunstoneArchiveDeployer;
-import sunstone.core.spi.SunstoneArchiveDeployerProvider;
-
-import java.lang.annotation.Annotation;
-import java.util.Optional;
-
-
-public class AzureArchiveDeployerProvider implements SunstoneArchiveDeployerProvider {
```

Comment:
Just idea. Isn't product agnostic way of deployment copying war/jar "somewhere" on azure. I saw sunstone is already dealing with ftp in some cases. Can't that be used?

Reply:
It depends on the resource. On VMs, there is no universal way. On App Services, partially yes, you can specify a gith repository. On the other hand appservice.deploy() (from Azure SDK) supports only WAR archive type for EAP stack and not JAR, nor EAR.